datatypes/graph.py

```diff
@@ -0,0 +1,21 @@
+"""Data classes used by our model"""
```

Comment:
Suggestion: Data classes for Graph Neural Network model.

Reply:
Let's describe here also the type of structure. That is, how the Graph class is defined with a list of features, senders and receivers for each point.